Modules/Microsoft365DSC/DSCResources/MSFT_EXOAntiPhishPolicy/MSFT_EXOAntiPhishPolicy.schema.mof

```diff
@@ -10,7 +10,7 @@ class MSFT_EXOAntiPhishPolicy : OMI_BaseResource
     [Write, Description("The TargetedUserProtectionAction parameter specifies the action to take on detected user impersonation messages for the users specified by the TargetedUsersToProtect parameter."), ValueMap{"BccMessage","Delete","MoveToJmf","NoAction","Quarantine","Redirect"}, Values{"BccMessage","Delete","MoveToJmf","NoAction","Quarantine","Redirect"}] String TargetedUserProtectionAction;
     [Write, Description("Specify if this policy should be enabled. Default is $true.")] Boolean Enabled;
     [Write, Description("The EnableFirstContactSafetyTips parameter specifies whether to enable or disable the safety tip that's shown when recipients first receive an email from a sender or do not often receive email from a sender.")] Boolean EnableFirstContactSafetyTips;
-    [Write, Description("The EnableAntispoofEnforcement parameter specifies whether to enable or disable antispoofing protection for the policy.")] Boolean EnableAntispoofEnforcement;
+    [Write, Description("DEPRECATED.")] Boolean EnableAntispoofEnforcement;
```

Comment:
Can we please add verbose in the Set-TargetResource to warn users that this is deprecated if they are using it. Thanks